tasks/deploy/config/mainnet/susds-182day.ts

```diff
@@ -0,0 +1,84 @@
+import { Address, keccak256, parseEther, toBytes, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import {
+    USDS_ADDRESS_MAINNET,
+    SUSDS_ADDRESS_MAINNET,
+    SIX_MONTHS,
+} from "../../lib/constants";
+import { MAINNET_ERC4626_COORDINATOR_NAME } from "./erc4626-coordinator";
+import { MAINNET_FACTORY_NAME } from "./factory";
+
+// The name of the pool.
+export const MAINNET_SUSDS_182DAY_NAME = "ElementDAO 182 Day sUSDS Hyperdrive";
+
+// The initial contribution of the pool.
+const CONTRIBUTION = parseEther("100");
+
+export const MAINNET_DAI_182DAY: HyperdriveInstanceConfig<"ERC4626"> = {
+    name: MAINNET_SUSDS_182DAY_NAME,
+    prefix: "ERC4626",
+    coordinatorAddress: async (hre) =>
+        hre.hyperdriveDeploy.deployments.byName(
+            MAINNET_ERC4626_COORDINATOR_NAME,
+        ).address,
+    deploymentId: keccak256(toBytes(MAINNET_SUSDS_182DAY_NAME)),
+    salt: toBytes32("0x69420"),
+    extraData: "0x",
+    contribution: CONTRIBUTION,
+    fixedAPR: parseEther("0.0625"),
+    timestretchAPR: parseEther("0.05"),
+    options: async (hre) => ({
+        extraData: "0x",
+        asBase: true,
+        destination: (await hre.getNamedAccounts())["deployer"] as Address,
+    }),
+    // Prepare to deploy the contract by setting approvals.
+    prepare: async (hre) => {
+        let baseToken = await hre.viem.getContractAt(
+            "contracts/src/interfaces/IERC20.sol:IERC20",
+            DAI_ADDRESS_MAINNET,
+        );
+        let tx = await baseToken.write.approve([
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_ERC4626_COORDINATOR_NAME,
+            ).address,
+            CONTRIBUTION,
+        ]);
+        let pc = await hre.viem.getPublicClient();
+        await pc.waitForTransactionReceipt({ hash: tx });
+    },
+    poolDeployConfig: async (hre) => {
+        return {
+            baseToken: DAI_ADDRESS_MAINNET,
```

Comment:
Should this and the below be the USDS addresses?

Reply:
yes